app/experimenter/docs/adrs/0003-analysis-visualization-architecture-approach.md

```diff
@@ -0,0 +1,49 @@
+# Analysis Visualization: Architecture Approach
+
+* Status: accepted
+* Author: Marina Samuel
+* Deciders: Nimbus Team
+* Date: 2020-09-16
+
+## Context and Problem Statement
+
+Analysis and visualization of experiment result data needs to be [part of the experiment console](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md). The mockups based on user interviews are [here](https://www.figma.com/file/q6Xh9g6LrOj2OQ59LbAFXm/Experiment-Dashboard-Mockup).
```

Comment:
This figma link may have changed, I can't see it and had it previously bookmarked so I don't think it's a permission thing for me🤔

Reply:
I also can't see the link

Reply:
Are you guys still unable to see the link? I think maybe figma was down for a bit yesterday, but it seems to work for me now?

Reply:
Yeah the link broke for me too, this works for me now: 

https://www.figma.com/file/5SLa07XqQb0ttB1ay1gami/Nimbus-Console-UI?node-id=214583%3A0

Reply:
The link in the ADR doesn't work for me still, but Jared's link does.

Reply:
So it looks like there are permissions issues because IT has changed things around. I'll link the Nimbus Console mockups for now and I'll be adding the dashboard mockups in there once we sort out permissions stuff.